compysition/actors/util/udplib.py

```diff
@@ -178,15 +178,12 @@ def run(self):
     
     def reap_peers(self):
         now = time.time()
-        for peer in list(self.peers.values()):
-            if peer.expires_at < now:
-                self.peers.pop(peer.uuid)
-                self.log("Reaping expired peer ({0}) from '{1}' peer pool. (Total remaining: {2})".format(peer.uuid, self.service, len(self.peers) + 1))
```

Comment:
We should probably keep this log message

Reply:
I understand wanting to keep it.  I was also hesitant to remove it.  However I can't see a way where we would not be sacrificing efficiency to have it in there.  What are your thoughts?

Reply:
Are there significant differences in efficiency? Worth losing a log message over?

Reply:
Really i don't think it would be the end of the world if we left it the same, though I honestly don't know how important that log message is?

The way I see it, least to most efficient, we can either leave it as it was, incorporate itervalue and a temp variable (with log), or use a dictionary comprehension (without log).